### ISSUE TITLE
The "Case" of the Missing CamelCase String Filter (Liquid/Jekyll)

### ISSUE THREAD

**Issue by evolutionxbox** [Shopify/liquid#755]
It was a grey, stormy Thursday afternoon. I was in the office, pondering Jekyll and Liquid templates. Suddenly. A sharp chill overcame me. I had found the `camelcase` filter (a.k.a `camelize`), to be missing presumed dead.

`{{ page.title | camelcase }}` did nothing and neither did `{{ page.title | camelize }}`.

Shocked, I furiously checked the [Shopify Liquid Templating documentation](https://help.shopify.com/themes/liquid/filters/string-filters#camelcase). 

There it was.

Confused, I checked the [Jekyll Filter documentation](https://jekyllrb.com/docs/templates/).

My eyes focused upon the phrase:

> "All of the standard Liquid tags and **filters** are supported."

Upon further investigation, I found that the Camelcase filter not only was not in [the standard list of filters](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters), it was [not even implemented](https://github.com/Shopify/liquid/blob/1e533a52e7368e34030163490aa72aa416c117ed/lib/liquid/standardfilters.rb)!

Feeling resigned to this discovery, I decided to plead on StackOverflow for help implementing a camelcase filter in ruby, as I do not know ruby.

---

### tl:dr;

The camelize/camelcase filter is missing in Jekyll and the Shopify Liquid implementation is not there either. Where is it?

**Comment by fw42** [comment]
The `camelize` filter is not a standard Liquid filter but a custom Shopify filter (your link points to the Shopify documentation, not the Liquid documentation).

**Comment by evolutionxbox** [comment]
Where is this filter defined?

**Comment by fw42** [comment]
In the shopify codebase

**Comment by fw42** [comment]
It's basically just a wrapper around [ActiveSupport's camelize method](http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-camelize). But we don't want to introduce that dependency in Liquid core, which is why it's not included here.

**Comment by evolutionxbox** [comment]
Thanks. I've been going up the wrong trees for a few hours now.
